source/Settings/View.cpp

```diff
@@ -56,14 +59,14 @@ namespace FileShare::GUI::Settings {
         this->addSubItem("Advanced");
 
         this->menuItems["Account"] = this->createAccountSettings();
-        this->addItem("assets/images/account_black.svg", "Account");
+        this->addItem("assets/images/account" + iconThemeSuffix + ".svg", "Account");
 
         auto logoutButton = Components::Button::create();
         logoutButton->setText("Logout");
         logoutButton->setType(Components::Button::Type::Danger);
-        logoutButton->onPress([=]() {
+        logoutButton->onPress([this]() {
             this->getSignal("logout").emit(this); // TODO: Implement signal. For now, it's not a bug; it's a feature
-        });
+            });
```

Comment:
Was good before - should be aligned with the begining of the line that opened them.